src/plugins/intel_cpu/src/nodes/interpolate.h

```diff
@@ -173,6 +192,7 @@ class Interpolate : public Node {
             int spatialDimSize;
             size_t dataRank;
             std::vector<int> indexTable;
+            std::vector<uint8_t> pillow_working_buf;
```

Comment:
As now in interpolate-11, indexTable is mainly used to store weights, can we call it auxTable or some other proper name, to avoid confusion?

Reply:
make sense, fixed, thanks